recipe/patches/0001-Fix-build-to-work-with-Anaconda-compiler-toolchain.patch

```diff
@@ -0,0 +1,39 @@
+From 6261d227b0cce2b765b3ca5e6dd661d6169c8f1e Mon Sep 17 00:00:00 2001
+From: Tim Snyder <snyder.tim@gmail.com>
+Date: Thu, 5 Mar 2020 17:17:17 -0600
+Subject: [PATCH] Fix build to work with Anaconda compiler toolchain
```

Comment:
Did you want this to be included in the patch?

Reply:
Meh.  I could take it out but I just used the output of `git format-patch` without modifying them.  Is that stuff hurting anything?

I'd eventually like to have this incorporated into the cmake files but used a patch for now since it is only needed when using the conda toolchains.

Reply:
Thanks @timsnyder incorporating the patch into the cmake files will help our local devs on RHEL7 and any on-premise CI flows too.